.github/workflows/codeql.yml

```
@@ -0,0 +1,37 @@
+name: "CodeQL"
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)